Dockerfile

```diff
@@ -1,4 +1,19 @@
-FROM alpine:3.11
+#Add qemu function for supporting multiarch
+ARG IMAGEARCH
+FROM alpine:3.12 as qemu
+RUN apk add --no-cache curl
+ARG QEMUVERSION=4.0.0
```

Comment:
🙋 This version of QEMU for multiple architectures [is over a year old](https://github.com/multiarch/qemu-user-static/releases/tag/v4.0.0) now. Could a more recent version be used instead? Even the v4.0.x series has had [v4.0.0-4](https://github.com/multiarch/qemu-user-static/tree/v4.0.0-4) released subsequently.